platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/Browser.java

```diff
@@ -628,8 +629,9 @@ private Collection<SwirldsPlatform> createLocalPlatforms(
 
         int ownHostIndex = 0;
 
-        for (final Address address : addressBook) {
-            final NodeId nodeId = address.getNodeId();
+        for (int nodeIndex = 0; nodeIndex < addressBook.getSize(); nodeIndex++) {
```

Comment:
Should `AddressBook` implement `Iterable<NodeId>`?

Reply:
It implements `Iterable<Address>` which contains the node Ids.  Now that NodeId is  non-contiguous, the concept of a NodeIndex in the range of [0 .. (N-1)] has become relevant in contrast.  The Node Index is used in places that relied on the NodeIds being contiguous starting from 0.  In the Browser, the GUI is tied to using logic based off of index for contiguous display of windows.

Reply:
no change made.